packages/sdk/src/flags.ts

```diff
@@ -52,7 +53,6 @@ export interface FlagResolver {
   // internal
   // evaluateFlagSync<T extends Value>(path: string, defaultValue: T): FlagEvaluation<T>;
 
-  evaluateFlag(path: string, defaultValue: number): FlagEvaluation<number>;
```

Comment:
@nicklasl we're actually gonna need these overloads for simple types, that's what saves us from getting the exact value inferred. Like `const v = c.evaluateFlag('a.b', 3)` Then v should have the type number rather than `3`.

Reply:
ack!